kusto/statemachine_test.go

```diff
@@ -407,6 +407,7 @@ func TestProgressive(t *testing.T) {
 							value.Long{Value: 0, Valid: false},
 						},
 					},
+					TableFragmentType: "DataReplace",
```

Comment:
Maybe add a test case to make sure that if we get multiple rows from a "DataReplace" table, the "Replace" flag should only be turned on for the first row